README.md

```diff
@@ -25,23 +25,32 @@ This repository adds support for the [Windows 10 SDK](https://developer.microsof
 
 See the official [React Native website](https://facebook.github.io/react-native/) for an introduction to React Native.
 
+## System Requirements
+
+- You can build and deploy React Native Windows apps on Windows 7 SP1, Windows 8.1, or Windows 10 
+- You can run React Native Windows UWP apps only on Windows 10 devices, but React Native Windows WPF apps will run on Windows 7-10 so long as .NET 4.6 is installed on the end user's machine
+- Download [Visual Studio 2015 Community](https://www.visualstudio.com/vs/older-downloads/)
+- - You will need to start Visual Studio after it is installed to do some final setup before it can be used to build or run your React Native Windows application
+- [Windows 10 SDK Build 14393](https://developer.microsoft.com/en-us/windows/downloads/sdk-archive)
+
+*Note*: Development on React Native Windows itself currently requires Visual Studio 2015. It is not supported with Visual Studio 2017 or with VS Code, but we will gladly accept pull requests to enable a great developer experience in those  environments.
+
 ## Getting Started
 
 - Follow the [Getting Started guide](http://facebook.github.io/react-native/docs/getting-started.html) to install React Native and its dependencies.
 - Change your working directory to the project directory you specified in the `react-native init` command from the [Getting Started guide](http://facebook.github.io/react-native/docs/getting-started.html) (e.g., `cd AwesomeProject`).
 - Install [rnpm-plugin-windows](https://www.npmjs.com/package/rnpm-plugin-windows) from NPM (i.e., `npm install --save-dev rnpm-plugin-windows`).
 - Initialize Windows project with `react-native windows` command in the project directory.
-- Learn more about various [tools for debugging React Native](http://facebook.github.io/react-native/docs/debugging.html).
+- *Note*: You will need to install the [requirements](#system-requirements) before you can build and run your application.
+- Open the solution file in the application folder in Visual Studio (e.g., `AwesomeProject/windows/AwesomeProject.sln`) 
+- *Note*: If this is your first time doing UWP development on the computer you're using, you may be asked to install additional UWP tooling. After opening the solution, right click the Solution in the Solution Explorer and select the option labeled "Install Missing Components". You may have to shutdown Visual Studio to continue the installation.
+- Select the "DebugBundle" configuration and the "x64" platform from the combo box controls to the left of the Run button and underneath the Team and Tools menu itema. 
```

Comment:
DebugBundle will require the user build a bundle prior to running the app. Instead, I would suggest running Debug and starting the server on the command line (a lot less intimidating in a first run scenario).